lib/client.js

```diff
@@ -44,10 +45,14 @@ export default class Client {
         }
 
         let data;
-        await fetch(fullUrl, options).then(response => {
+        await fetch(fullUrl, options).then(async (response) => {
             if (response.ok) {
                 if (method === 'get') {
-                    data = response.text();
+                    data = await response.text();
```

Comment:
I guess we should also use the `await` statement in the line 60 now (`post` block)

Reply:
@lovrocolic I guess its not required. Since either way function is returning a promise at the end.